providers/nedb.js

```diff
@@ -0,0 +1,82 @@
+let db;
+let Datastore;
+
+const config = {
+  persistent: false,
+};
+
+exports.init = (client) => {
```

Comment:
You took the advantage of Download v2 being able to recognize `require` outside module exports in the nedb-class provider.